_includes/lines.html

```diff
@@ -0,0 +1,3 @@
+<script type="text/javascript" src="/js/gsap.1.19.0.js"></script>
```

Comment:
Are we using GSAP?  I don't see where in the code below.

Reply:
Hello, yes we have a timeline in 152 + Elastic ease used

Reply:
I wonder if we really need two libraries?  Paper.js can't do what we need on its own?

Reply:
I didnt find any example with this level of physics for thoses line, its crazy